### PR TITLE
My VA - downtime message for VBA maintenance

### PR DESCRIPTION
**Pull request by allisonlu**
## Summary

This PR adds a downtime message (`va-alert` web-component) to the My VA page when VBA performs maintenance for COLA. This temporary update lives behind the `authExpVbaDowntimeMessage`/`auth_exp_vba_downtime_message` feature flag.

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/69472

- PR for feature flag implementation (on Platform level): https://github.com/department-of-veterans-affairs/vets-website/pull/26694
- PR on `vets-api`: https://github.com/department-of-veterans-affairs/vets-api/pull/14494

## Testing done
- locally, visually
- all unit and e2e (?) tests still pass

## Screenshots 

 | Before | After |
 | ------ | ----- |
  |   downtime-msg-before  |  downtime-msg-after   |

## What areas of the site does it impact?
My VA - entire page

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by powellkerry — APPROVED** [review]

File: src/applications/personalization/dashboard/components/Dashboard.jsx [hunk 1]
```diff
@@ -190,6 +195,14 @@ const Dashboard = ({
     [canAccessPaymentHistory, getPayments],
   );
 
+  // use session storage to track if downtime alert has been dismissed
+  const [dismissed, setDismissed] = useSessionStorage(
+    'authExpVbaDowntimeMessageDismissed',
```
Comment (adamwhitlock1) [inline src/applications/personalization/dashboard/components/Dashboard.jsx:200]: Should we use a different key for My VA vs Profile for this?

**Review by adamwhitlock1 — APPROVED** [review]
Not sure about the session storage key if it should be different or not, but otherwise LGTM

**Review by adamwhitlock1 — APPROVED** [review]
👍🏼

File: src/applications/personalization/dashboard/components/Dashboard.jsx [hunk 2]
```diff
@@ -247,29 +260,73 @@ const Dashboard = ({
...
+                        </p>
+                      </div>
+                    </VaAlert>
+                  </div>
+
+                  {isLOA3 && (
+                    <>
+                      <HealthCare isVAPatient={isVAPatient} />
+                      <EducationAndTraining />
+                      <SavedApplications />
```
Comment (adamwhitlock1) [inline src/applications/personalization/dashboard/components/Dashboard.jsx:296]: Did saved applications get removed from the imports?

Comment (allisonlu) [reply src/applications/personalization/dashboard/components/Dashboard.jsx]: yes :( just pushed up a fix

**Pull request merged**